client/src/components/DeviceSelector.tsx

```diff
@@ -2,9 +2,9 @@ import React, { useEffect, useContext } from "react";
 import Select from "react-select";
 import PropTypes from "prop-types";
 import { Device, listInputDevices, selectInputDevice } from "./DeviceSelectorHelper";
```

Comment:
Is `DeviceSelectorHelper` left out because it's a helper in the same directory, or should it use the absolute path too?